STYLE-GUIDE.md

```diff
@@ -19,10 +19,28 @@ Words are important.  Pulumi strives to use language that is clear, harmonious,
 * Ensure that readers are able to scan the headings of a page and get an effective overview of the page's content.
 * Every page should have exactly one `h1`.
 * Headings levels should only increment one level at a time.  E.g., if your previous heading level was an `h2`, the next heading must be an `h2` or an `h3`, but not, e.g., an `h4` or `h5`.
+*  Include keywords relating to the content immediately following the heading in `h2` and deeper.
+*  Try to front-load your keywords into the sentence to more immediately indicate the subject matter of the following content.
+*  - Alternate your topical keywords, visit the [online Thesarus](https://www.thesaurus.com/) if you are stuck.
```

Comment:
In Markdown, you indent with spaces to get a sub-point in a list. Also, a spelling fix and a punctuation fix.

```suggestion
  * Alternate your topical keywords; visit an [online thesaurus](https://www.thesaurus.com/) if you are stuck.
```

As an aside, as an editor, this kind of behavior is something I discourage. We don't want to sound like we're in school trying to sound smart. Suggesting someone should try to use multiple keywords is good. Suggesting we use a thesaurus to vary as much as possible sounds odd.